executor/executor_test.go

```diff
@@ -185,6 +185,18 @@ func (s *testSuite) TestAdmin(c *C) {
 	c.Assert(row.GetString(1), Equals, historyJobs[0].State.String())
 	c.Assert(err, IsNil)
 
+	// show DDL job queries test
+	tk.MustExec("use test")
+	tk.MustExec("create table admin_test2 (c1 int, c2 int, c3 int default 1, index (c1))")
+	result := tk.MustQuery(`admin show ddl job queries 43`)
```

Comment:
Is it unstable when we use `43`?

Reply:
New add test ddl will affect.

Reply:
New added ddl test job id incremented.
In accordance with the order and test is very stable.